core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -63,54 +63,72 @@ internal class CacheFallbackOperations
                 return;
             }
 
-            var rtItem = new MsalRefreshTokenCacheItem
-                (new Uri(authority).Host, clientId, resultWrapper.RefreshToken, resultWrapper.RawClientInfo);
-
-            tokenCacheAccessor.SaveRefreshToken(rtItem);
-
-            MsalAccountCacheItem accountCacheItem = new MsalAccountCacheItem
-                (new Uri(authority).Host, objectId, resultWrapper.RawClientInfo, null, displayableId, resultWrapper.Result.TenantId);
+            try
+            {
+                var rtItem = new MsalRefreshTokenCacheItem
+                    (new Uri(authority).Host, clientId, resultWrapper.RefreshToken, resultWrapper.RawClientInfo);
+                tokenCacheAccessor.SaveRefreshToken(rtItem);
 
-            tokenCacheAccessor.SaveAccount(accountCacheItem);
+                MsalAccountCacheItem accountCacheItem = new MsalAccountCacheItem
+                    (new Uri(authority).Host, objectId, resultWrapper.RawClientInfo, null, displayableId, resultWrapper.Result.TenantId);
+                tokenCacheAccessor.SaveAccount(accountCacheItem);
+            }
+            catch (Exception ex)
+            {
+                msg = "An error occurred while writing ADAL refresh token to the cache in MSAL format. " +
+                      "For details please see https://aka.ms/net-cache-persistence-errors. ";
```

Comment:
>net-cache-persistence-errors [](start = 61, length = 28)

Jean-Marc suggested: 
msal-net-cache-persistance-errors 
for naming consistency.

Reply:
based on conversation today that the message is used across libraries. approving